package SyncDemo;

message ChatMessage {
  required string to = 1;
  required string from = 2;
  enum ChatMessageType {
    CHAT = 0;
    HELLO = 1;
    LEAVE = 2;
    OTHER = 3;
  }
  required ChatMessageType type = 3 [default = CHAT];
  optional string data = 4;
  required int32 timestamp = 5;
}
